scripts/gencert.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Generate xapi ssl certificate
+Requires=xapi.service forkexecd.service xapi-init-complete.target
+After=xapi.service forkexecd.service xapi-init-complete.target
+
+[Service]
+Type=oneshot
+ExecStart=/opt/xensource/libexec/gencert /etc/xensource/xapi-ssl.pem
+RemainAfterExit=yes
```

Comment:
Does the default of `no` introduce problems? I thought this was needed in case the service forked and systemd lost visibility on it but still needs to stop it.

Reading the docs it seems the default should be OK in this case for dependency purposes:
```
Behavior of oneshot is similar to simple; however, the service manager will consider the unit up after the main
               process exits. It will then start follow-up units.
```

Reply:
i doubt it would cause problems, but if we set `RemainAfterExit=no`, then systemd will run the gencert binary each time we start/restart stunnel which isn't necessary

Reply:
the firstboot changes use `RemainAfterExit=yes` also https://github.com/xapi-project/xen-api/pull/4027/files